src/vaadin-time-picker.html

```diff
@@ -619,6 +624,7 @@
           return TimePickerElement.properties.i18n.value().parseTime(text);
         }
 
+        /** @private */
         _getInputElement() {
           return this.shadowRoot.querySelector('vaadin-time-picker-text-field');
         }
```

Comment:
I wasn't sure what to do with this. I feel like this `_getInputElement` should be private and the `__inputElement` getter below should be protected, but their number of underscores implies the exact opposite. So now they're both private.